src/storage/SplitCache/killLocally.js

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Why is this not part of the storage? Either we make it part of the Split caches (where each one requires it's own handling) or we take this functionality, ideally, at the same place where we're executing this and passing the storage down.

The way it is today it's tricky because you would need to handle the asynchronicity of the getSplit and the addSplit methods here, would be better if we move it to each storage (you're taking advantage of the fact that the only async storage we have today is Redis where we don't sync the data, but that's changing in the next 6 months)

Reply:
refactored: killLocally was added to the prototypes of Split caches.